### ISSUE TITLE
eth.getTransactionReceipt: contractAddress: Error: Cannot access member 'constructor' of null

### ISSUE THREAD

**Issue by jorisbontje** [ethereum/go-ethereum#1612]
with `geth 1.0.1` in the JS console (via `geth attach`):

```
> eth.getTransactionReceipt('0x9d3f3e5b651c3b4472eca4cd57db6549d474a23af182c768d418bd9e278195b5')
{
  blockHash: '0x35803048a469e86a0e9df1641ddf84a237aad0b4c0702ee9b99e1035afc992f0',
  blockNumber: 47527,
  contractAddress: Error: Cannot access member 'constructor' of null,
  cumulativeGasUsed: 21000,
  gasUsed: 21000,
  logs: [ ],
  transactionHash: '0x9d3f3e5b651c3b4472eca4cd57db6549d474a23af182c768d418bd9e278195b5',
  transactionIndex: 0
}
```

I don't expect to see the error message `contractAddress: Error: Cannot access member 'constructor' of null,`, but probably `null`, `0x` or something similar if no `contractAddress` is involved.

**Comment by obscuren** [comment]
Output error. Will be fixed, thanks.

**Comment by jorisbontje** [comment]
Seems fixed in `1.1.1`
